File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for(int i = 0; i < nums.length; i++){
            //System.out.println(i);
            for(int j = 0; j < nums.length; j++){
                if( j == i){
                    continue;
                }

                if ((nums[i] + nums[j]) == target){
                    //System.out.println(i);
                    //System.out.println(j);
                    int result[] = {i, j};
                    return result;
                }
            }
        }
        //System.out.println(nums);
        return nums;
    }
}

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if j == i:
                    continue
                else:
                    if (nums[i] + nums[j]) == target:
                        ans = [i, j]
                        break
            if ans != []:
                break
        return ans
            
        


